packages/main/src/plugin/install/extension-installer.ts

```
@@ -116,17 +134,38 @@ export class ExtensionInstaller {
     imageName: string,
     catatlogExtensionId?: string,
     onProgress?: (progress: number) => void,
+    token?: CancellationToken,
   ): Promise<AnalyzedExtension | DockerDesktopContribution | undefined> {
     imageName = imageName.trim();
     sendLog(`Analyzing image ${imageName}...`);
-    let imageConfigLabels;
+    let localImage;
```

Biome (2.5.7)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)